### PR TITLE
fix(DB/Dungeon/Quest): The Codex of Blood

### PR DESCRIPTION
**Pull request by wizzymore**
## CHANGES PROPOSED:
- Make The Codex of Blood to be seen in heroic as well

## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3032

## TESTS PERFORMED:
- Checked the GObject in normal and heroic and is working as intended

## HOW TO TEST THE CHANGES:

Change dificulty to normal first, and then test again on heroic, it should appear on both normally
.go c 69955
Take the quest The Codex of Blood ( https://tbcdb.com/?quest=10094 )
Make sure you are level 70 ( if not, set your level to 70 and relog to see the quest )
.go gob 23860

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Review by Helias — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1601868704763174600.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1601868704763174600');
+
+DELETE FROM `gameobject` WHERE (`id` = 182947) AND (`guid` IN (23860));
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1601868704763174600.sql:3]: this query may fail if there is already a row having `guid` = `23860` and `id` different than `182947`.

If you want to make sure the query doesn't fail, then:

```sql
DELETE FROM `gameobject` WHERE `guid` = 23860;
```

Also, if (and ONLY if) you want to make sure your new `INSERT` will be the ONLY spawn of the gameobject `182947`, you can add this too:

```sql
DELETE FROM `gameobject` WHERE `id` = 182947;
```

resulting in:

```sql
DELETE FROM `gameobject` WHERE `guid` = 23860;
DELETE FROM `gameobject` WHERE `id` = 182947; -- only if you want to delete all current spawns of 182947
```

**Review by FrancescoBorzi — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1601868704763174600.sql (hunk 1 above)
Comment (wizzymore) [reply data/sql/updates/pending_db_world/rev_1601868704763174600.sql:3]: Would it be better to have something like this?
```sql
 UPDATE `gameobject` SET `spawnMask` = 3 WHERE (`guid` = 23860 and `map` = 555);
```
As we only need the spawnMask of 23860 to be updated in the map 555 (Auchindoun: Shadow Labyrinth).

FYI: The generated query i added in the `.sql` was generated by Keira3

Comment (FrancescoBorzi) [reply data/sql/updates/pending_db_world/rev_1601868704763174600.sql]: ah yes, definitely. If you are updating an existing row, always prefer `UPDATE`

Comment (wizzymore) [reply data/sql/updates/pending_db_world/rev_1601868704763174600.sql]: Perfect :D

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by Gierlappen** [comment]
Tested here https://github.com/azerothcore/azerothcore-wotlk/issues/3032

**Review by Gierlappen — APPROVED** [review]

**Comment by Racknar0** [comment]
Tested! 

image

I doubt that the Gobject was in the air but check and your position is correct

image

**Pull request merged**